playbooks/aws/terminate.yml

```diff
@@ -9,7 +9,8 @@
 
 - name: Gather dynamic inventory variables for hosts to terminate
   hosts: oo_hosts_to_terminate
-  gather_facts: no
+  roles:
+  - rhel_unsubscribe
```

Comment:
I would probably suggest that we switch this to a conditional role, like the subscribe role below.